src/app/components/Sidebar/constants.js

```diff
@@ -0,0 +1,22 @@
+export const CATEGORIES = [
+  {
+    id: 0,
+    name: 'BUTTONS'
```

Comment:
The names will be in lowercase so we should write them like that in here and transform to uppercase with CSS